data/bhawanak0504

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "bhawana0504",
+  "type": "community",
+  "bio": "open source community",
+   "links": [
+    {
+      "name": "Follow me on github",
+      "url": "https://github.com/bhawana0504",
+      "icon": "Fagithub"
```

Comment:
This is not one of the approved icons.  I would suggest you take a look at: https://linkfree.io/icons

I expect you may want `FaGithub` - please note these are case sensitive.